tests/test_views.py

```diff
@@ -151,6 +151,34 @@ def test_jwt_login_using_zero(self):
 
         self.assertEqual(response.status_code, 400)
 
+    def test_jwt_permanent_token_auth(self):
+        api_settings.JWT_PERMANENT_TOKEN_AUTH = True
+
+        client = APIClient(enforce_csrf_checks=True)
+        client.credentials(HTTP_X_DEVICE_MODEL='Nokia', HTTP_USER_AGENT='agent')
+        self.assertEqual(Device.objects.all().count(), 0)
+        response = client.post('/auth-token/', self.data, format='json')
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
```

Comment:
Please also show what is returned in response. I see that device was created, but I have no idea if the jwt token or device id was returned.